.github/workflows/close_stale_pull_requests.yml

```diff
@@ -18,11 +18,8 @@ jobs:
           # Do not handle issues at all. We only want to handle PRs.
           days-before-issue-stale: -1
 
-          stale-pr-message: "This PR is stale because it has been open for 30 days with no activity. Remove stale label or comment, or this will be closed in 7 days."
+          stale-pr-message: "This PR is being marked as stale because it has been open for 30 days with no activity. Please remove the stale label or add a comment, or it will be closed in 7 days."
```

Comment:
(1) Does removing the `Stale` label count as activity and reset the timer?
(2) Do Draft PRs also get closed?

Reply:
Depending on the answer to (2) above, I would refine this message a bit further:
```
This PR is being marked as stale because it has been open for 30 days with no activity. You may take one of the following actions:
- Change it to a Draft PR
- Remove the stale label
- Add a comment stating why it is not stale
If no action is taken, it will be closed in 7 days.
```

Reply:
> Does removing the Stale label count as activity and reset the timer?

Yep, that's my understanding. Anything that changes the `updated_at` field (which happens for basically anything set in the [body params](https://docs.github.com/en/rest/issues/issues#update-an-issue)).

> Do Draft PRs also get closed?

In our current configuration, yes. There's a bit about draft PRs consuming way more operations due the limitations of the GitHub API, described [here](https://github.com/actions/stale#exempt-draft-pr)

Reply:
I'm good with Draft PRs being closed :)
Can you change the wording to the following?
```
This PR is being marked as stale because it has been open for 30 days with no activity. You may take one of the following actions:
- Remove the stale label
- Add a comment stating why it is not stale
If no action is taken, it will be closed in 7 days.
```

Reply:
yep! there's one other action you can take — namely, "push additional commits to this branch" — so i'll include that as well